synapseclient/core/upload/multipart_upload.py

```diff
@@ -412,28 +420,6 @@ def __call__(self):
         return upload_status_response
 
 
-def _get_file_chunk(file_path, part_number, chunk_size):
```

Comment:
I pulled out common functionality out of the previous upload scripts that could be re-used. I placed them into `upload_utils.py`